python/ray/scripts/scripts.py

```diff
@@ -341,121 +305,80 @@ def dashboard(cluster_config_file, cluster_name, port, remote_port):
     help="manually specify the socket path of the raylet process")
 @click.option(
     "--temp-dir",
+    hidden=True,
     default=None,
     help="manually specify the root temporary dir of the Ray process")
 @click.option(
     "--java-worker-options",
     required=False,
+    hidden=True,
     default=None,
     type=str,
     help="Overwrite the options to start Java workers.")
 @click.option(
     "--code-search-path",
     default=None,
+    hidden=True,
```

Comment:
This option is now documented. We should not hide it, right?

Reply:
thanks